app/lib/account/models.dart

```diff
@@ -63,6 +66,68 @@ class OAuthUserID extends db.ExpandoModel {
   String get userId => userIdKey.id as String;
 }
 
+/// Maps the session id (from cookie) to User.id and cached profile properties.
+class UserSession extends db.ExpandoModel {
+  /// Same as [id].
+  /// This is a v4 (random) UUID String.
+  String get sessionId => id as String;
+
+  @db.ModelKeyProperty(required: true)
+  db.Key userIdKey;
+
+  String get userId => userIdKey.id as String;
+
+  @db.StringProperty(required: true)
+  String email;
+
+  @db.StringProperty()
+  String imageUrl;
+
+  @db.DateTimeProperty(required: true)
+  DateTime created;
+
+  @db.DateTimeProperty(required: true)
+  DateTime expires;
+
+  bool isExpired() => DateTime.now().isAfter(expires);
+}
+
+/// The cacheable version of [UserSession].
+@JsonSerializable()
+class UserSessionData {
+  final String sessionId;
+  final String userId;
```

Comment:
What is this? Can we document some of these... just a single comment for each.. 

SessionId is a random UUIDv4, and stuff like that... In particular highlight if they can be `null` or not..